Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -370,6 +371,11 @@ public struct ConvertAction: Action, RecreatingContext {
         
         // If we're building a navigation index, finalize the process and collect encountered problems.
         if let indexer = indexer {
+            let finalizeNavigationIndexMetric = benchmark(begin: Benchmark.Duration(id: "finalize-navigation-index"))
```

Comment:
I think we're already measuring this actually:

https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocCUtilities/Action/Actions/Convert/Indexer.swift#L85

Reply:
Actually- since we're manually creating the benchmark duration for the navigator creation, it looks like we're missing a `* 1000` that the regular benchmarks get...

https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Benchmark/Metrics/Duration.swift#L57

Reply:
I'll remove this and fix the existing metric.

Reply:
Since the existing metric was accumulating the time of parallel tasks I did removed it pinfavor of the new metric instead. This was done in 2f92635d573f36575fb22130c199d64e16749224